examples/message-ubench/main.pony

```diff
@@ -132,9 +138,29 @@ actor SyncLeader
     _env = env
     _initial_pings = initial_pings
     _ps = ps'
+
+    // Create timers
+    _timers = Timers
+
+    // Create the interval timer which will be cause the pinging
+    let interval: U64 = (report_interval.u64() * 1_000_000_000) / 10
+    let interval_timer = Timer(Tick(env, this), interval, interval)
+    _interval_timer = interval_timer
+    _timers(consume interval_timer)
+
+    // Get current time
     (let t_s: I64, let t_ns: I64) = Time.now()
+
+    if run_time > 0 then
```

Comment:
I'm not sure we need another timer.

I'd rather see the Tick timer know at what point it should stop and return false rather than true from its `apply` method.

Reply:
Yes, I thought of that but wanted to explore using timers and especially canceling. This is more asynchronous and thus likely to test more corner cases.

Reply:
They don't have to. Having tick stop firing based on time would work without the need for another timer

Reply:
I understand, but I like having two timers. For instance what if we randomized the Tick interval or due to a bug it stopped firing.

I'll add a comment indicating that RunTimeNotify is unnecessary, but if you insist I'll change it.